.github/workflows/spherical_geometry.yml

```diff
@@ -0,0 +1,48 @@
+name: Spherical Geometry CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Modify these like the other PRs so that all PR branches get built, not just those against `master`.  Also pushed tags.

Reply:
I made these changes.